sdk/keyvault/azure-security-keyvault-certificates/pom.xml

```diff
@@ -103,4 +103,20 @@
     </dependency>
 
   </dependencies>
+  <profiles>
+    <profile>
+      <id>java-lts</id>
+      <activation>
+        <jdk>[11,)</jdk>
+      </activation>
+      <dependencies>
+        <dependency>
```

Comment:
Why does this dependency need to be added to KeyVault specifically? Isn't this a core thing, why is KeyVault special?

Reply:
@mitchdenny I think this may be for the additional test dependency however there's a discussion above in the sdk/appconfiguration/azure-data-appconfiguration/pom.xml changes.